controller/bindings/connman/connman.ml

```diff
@@ -328,6 +328,9 @@ struct
     ; "name", s.name |> Ezjsonm.string
     ; "favorite", s.favorite |> Ezjsonm.bool
     ; "connected", s |> is_connected |> Ezjsonm.bool
+    ; "strength", (match s.strength with
+      | Some s -> string_of_int s ^ "%"
```

Comment:
Are you sure that this value is in percent? Just wondering because all networks listed seem rather low in strength.

Reply:
Yes, it’s in percent. Moving to nearer to the Wifi access point gives me this:

![2019-11-11_12-07-41](https://user-images.githubusercontent.com/6768842/68582935-39c6b100-047c-11ea-9fc8-32045e3c7c85.png)